sbt-plugin/src/sbt-test/cross-version/2.13/src/main/scala/sbttest/TestApp.scala

```diff
@@ -0,0 +1,12 @@
+package sbttest.noDOM
+
+object TestApp {
+
+  def main(args: Array[String]): Unit = {
+    println(Lib.foo("Hello World"))
+    println(Lib.sq(10))
+  }
+
+  // Used as a manual module initializer.
+  def foo(): Unit = ()
```

Comment:
This is unused. (it should also not appear in the 2.11 `TestApp.scala`, but that should be removed in a previous commit already (see my other comment on the topic).